### ISSUE TITLE
Components without props

### ISSUE THREAD

**Issue by annachu** [SoftwareBrothers/better-docs#26]
I'm really enjoying better-docs - thanks so much for making it!

Is it possible to include components that have no props? For example, the following component does not appear in the generated docs without PropTypes.

```
/**
 * Some documented component
 * 
 * @component
 */
const Documented = () => {
  return (
    <div>Hello world</div>
  )
}

export default Documented
```

Thanks!

**Comment by wojtek-krysiak** [comment]
It should work out of the box. I've added that component to example documentation here: https://softwarebrothers.github.io/example-design-system/Documented.html

Maybe your component doesn't have .jsx extension (base on that I verify if the component is vue or react) or you don't have jsx in source.includePattern section of your jsdoc.json file (it should be "includePattern": "\\.(vue|jsx)$")

**Comment by annachu** [comment]
Thanks for including the example. It's funny that this is still not working for me...

When I use the "Documented" component from your Readme, it works as expected and appears in the generated docs. But when I replace that code (in the same file) with the prop-less version of the component, the component does not appear in the final docs.

I've checked that I'm running the latest version (v1.3.3). The only other thing I can think of is that I'm using Preact (with an alias to map it to React)...but the small difference in the "Documented" component code shouldn't be affected by this.

If it helps, I've included [this test repo](https://gitlab.com/achu/better-docs-temp) that replicates the problem I'm having. You'll see that only the component with props is being picked up.

Thanks again for your help!
